tools/windows-node-installer/pkg/cloudprovider/aws/ec2.go

```diff
@@ -40,7 +40,7 @@ const (
 	awsPasswordDataTimeOut = 15 * time.Minute
 	// sshPort to access the OpenSSH server installed on the windows node. This is needed
 	// for our CI testing.
-	sshPort = 22
+	sshPort = types.SshPort
```

Comment:
This is unnecessary change again. Either update the entire code to use `types.SshPort` don't bring this change in.

Reply:
I agree with @ravisantoshgudimetla. I had clearly called out that these sort of changes need to be done in a separate pre-commit to the main one.